docs/topics/jvm/jvm-records.md

```diff
@@ -0,0 +1,74 @@
+[//]: # (title: Records)
+
+> JVM records feature is [Experimental](components-stability.md). It may be dropped or changed at any time. Use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-42430).
+>
+{type="warning"}
+
+_Records_ are Java specific classes for storing immutable data. Records carry a fixed set of values – the _records components_.
+As compared to regular Java classes, they have a concise syntax in Java and save you from writing boilerplate code:
+
+```java
+public record Person (String name, int age) {}
```

Comment:
If a page contains samples in both Kotlin and Java, please start Java snippets with the `// Java` comment as we do in [Calling Kotlin from Java](https://kotlinlang.org/docs/java-to-kotlin-interop.html), for example.

Reply:
Fixed, thx!